src/phoenix/core/traces.py

```diff
@@ -38,10 +38,26 @@ def get_project(self, project_name: str) -> Optional["Project"]:
         with self._lock:
             return self._projects.get(project_name)
 
-    def get_projects(self) -> Iterator[Tuple[str, "Project"]]:
+    def get_projects(self) -> Iterator[Tuple[int, str, "Project"]]:
         with self._lock:
-            projects = tuple(self._projects.items())
-        yield from projects
+            for project_id, (project_name, project) in enumerate(self._projects.items()):
+                if project.is_archived:
+                    continue
+                yield project_id, project_name, project
+
+    def archive_project(self, id: int) -> Optional["Project"]:
```

Comment:
nit. conflicts with built-in function `id()`
```suggestion
    def archive_project(self, id_: int) -> Optional["Project"]:
```